pom.xml

```diff
@@ -680,25 +677,6 @@ THE SOFTWARE.
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-enforcer-plugin</artifactId>
         <executions>
```

Comment:
@oleg-nenashev @jglick So this effectively removed the check I added in https://github.com/jenkinsci/jenkins/pull/781 for I believe still good reasons. 

Initially, I was assuming this was done for Java 11 related issues after I found this through bisection, but looking at the  date, this was done long before. So looking at the commit log, I assume this was unintended?

Reply:
:man_facepalming: OK, I missed it the first time. AIUI, it's been moved in the parent pom, so should still be running. NVM